### ISSUE TITLE
Release 1.3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7237]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#607|
|Type|defect|
|Created|2004-04-24 09:02:59Z|

<details><summary>Searchable metadata</summary>

```
trac-id__607 607
type__defect defect
reporter__itamarst itamarst
priority__highest highest
milestone__ 
branch__None None
branch_author__None None
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1082797379000000 1082797379000000
changetime__1480596037150707 1480596037150707
version__ 
owner__ 
cc__radix cc__exarkun cc__itamarst cc__hypatia cc__glyph
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
After:

1. Finishing news splitoff, buildbot support for that etc..
2. New site showing split off projects.
3. Finish remaining relevant bugs.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

In changeset 5cda954e9613d7ecd9a1f21e8f760a654571a7aa
```
#!CommitTicketReference repository="" revision="5cda954e9613d7ecd9a1f21e8f760a654571a7aa"
Merge pull request #607 from moshez/8919-procmon-py35-fix

Author: moshez

Reviewer: wsanchez

Fixes: #8919

On Python 3, procmon now handles process output without exceptions
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

Replying to [Glyph <glyph@…>](#note_3):
> In changeset 5cda954e9613d7ecd9a1f21e8f760a654571a7aa
> ```
> #!CommitTicketReference repository="" revision="5cda954e9613d7ecd9a1f21e8f760a654571a7aa"
> Merge pull request #607 from moshez/8919-procmon-py35-fix
> 
> Author: moshez
> 
> Reviewer: wsanchez
> 
> Fixes: #8919
> 
> On Python 3, procmon now handles process output without exceptions
> ```

Oops?  Heh.
